### PR TITLE
tabblock w/o children fix

### PR DESCRIPTION
**Pull request by Harleymckee**

### CODE REVIEW

File: app/assets/javascripts/revelry/ui/tabs/TabBlock.js.cjsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ Rev.registerComponent 'TabBlock',
       "RevTabBlock": true
       content: true
       active: @props.active
-    <div className={className} key={@props.key}>
-      {@props.children}
-    </div>
+    if @props.children
+      <div className={className} key={@props.key}> { @props.children } </div>
+    else
+      <div></div>
```
Comment (Dacello) [inline app/assets/javascripts/revelry/ui/tabs/TabBlock.js.cjsx]: if you need to render an empty element you can just do `<div />`

**Comment by Dacello** [comment]
You are gonna need to bump the gem version too

File: app/assets/javascripts/revelry/ui/tabs/TabBlock.js.cjsx (hunk 1 above)
Comment (Dacello) [inline app/assets/javascripts/revelry/ui/tabs/TabBlock.js.cjsx]: Also I think you should do probably do `if @props.children?` as it accounts for it being null or undefined

File: app/assets/javascripts/revelry/ui/tabs/TabBlock.js.cjsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ Rev.registerComponent 'TabBlock',
       "RevTabBlock": true
       content: true
       active: @props.active
-    <div className={className} key={@props.key}>
-      {@props.children}
-    </div>
+    if @props.children?
+      <div className={className} key={@props.key}> { @props.children } </div>
```
Comment (prehnRA) [inline app/assets/javascripts/revelry/ui/tabs/TabBlock.js.cjsx]: Format things:

* Wrap this onto multiple lines like it was before.
* No spaces after `{` or before `}`

File: app/assets/javascripts/revelry/ui/tabs/TabBlock.js.cjsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ Rev.registerComponent 'TabBlock',
...
+    else
+      <div />
```
Comment (prehnRA) [inline app/assets/javascripts/revelry/ui/tabs/TabBlock.js.cjsx]: Should we add classes and key to this also?

Comment (Dacello) [inline app/assets/javascripts/revelry/ui/tabs/TabBlock.js.cjsx]: definitely at least a key

Comment (Harleymckee) [inline app/assets/javascripts/revelry/ui/tabs/TabBlock.js.cjsx]: on the empty?

Comment (Dacello) [inline app/assets/javascripts/revelry/ui/tabs/TabBlock.js.cjsx]: yea

**Pull request merged**